File: src/main/java/logan/pickpocket/commands/AdminCommand.java
package logan.pickpocket.commands;

import logan.api.command.BasicCommand;
import logan.api.command.SenderTarget;
import logan.pickpocket.config.MessageConfiguration;
import logan.pickpocket.user.PickpocketUser;
import org.bukkit.entity.Player;

import java.util.List;

public class AdminCommand extends BasicCommand<Player> {

    public AdminCommand() {
        super("admin", "pickpocket.admin", 1, 2,
                new String[0], SenderTarget.PLAYER, "pickpocket",
                List.of(String.class, String.class),
                "/pickpocket admin bypass [name]\n" +
                "/pickpocket admin exempt [name]\n" +
                "/pickpocket admin notify");
    }

    @Override
    public boolean run(Player sender, String[] args, Object data) {
        PickpocketUser user = PickpocketUser.get(sender);
        user.setAdmin(!user.isAdmin());
        sender.sendMessage(MessageConfiguration.getAdminStatusChangeMessage(user.isAdmin()));
        return false;
    }
}

File: src/logan/pickpocket/commands/AdminCommand.kt
package logan.pickpocket.commands

import logan.api.command.BasicCommand
import logan.api.command.SenderTarget
import logan.pickpocket.config.MessageConfiguration
import logan.pickpocket.user.PickpocketUser
import org.bukkit.entity.Player

class AdminCommand : BasicCommand<Player>(
    "admin",
    argRange = 1..2,
    usage = """
        /pickpocket admin bypass [name]
        /pickpocket admin exempt [name]
        /pickpocket admin notify
    """.trimIndent(),
    argTypes = listOf(String::class, String::class),
    parentCommand = "pickpocket",
    target = SenderTarget.PLAYER,
    permissionNode = "pickpocket.admin"
) {
    override fun run(sender: Player, args: Array<out String>, data: Any?): Boolean {
        val profile = PickpocketUser.get(sender)
        var value = profile.playerConfiguration.adminSectionValue
        PickpocketUser.get(sender).playerConfiguration.setAdminSection(!value.also { value = it })
        sender.sendMessage(MessageConfiguration.getAdminStatusChangeMessage(value))
        return false
    }
}